spring-thymeleaf/pom.xml

```diff
@@ -78,6 +78,11 @@
 			<artifactId>thymeleaf-layout-dialect</artifactId>
 			<version>${thymeleaf-layout-dialect.version}</version>
 		</dependency>
+		<dependency>
+    		<groupId>org.thymeleaf.extras</groupId>

```

Comment:
Correct the indentation of this dependency, and extract the version number into a property.